core/client/fs/src/test/java/alluxio/client/file/cache/store/LocalPageStoreTest.java

```diff
@@ -0,0 +1,50 @@
+package alluxio.client.file.cache.store;
+
+import static org.junit.Assert.assertEquals;
+
+import alluxio.client.file.cache.PageStore;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.channels.WritableByteChannel;
+
+public class LocalPageStoreTest {
```

Comment:
test delete and other conditions?

Reply:
I will eventually add more tests in a follow-up PR